cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -406,6 +406,41 @@ macro(_ov_find_intel_gpu_dependencies)
     unset(_OV_ENABLE_ONEDNN_FOR_GPU)
 endmacro()
 
+macro(_ov_find_intel_npu_dependencies)
```

Comment:
I suppose it's not required, because we install target `ze_loader` and it's always defined when NPU is being built

Reply:
We export ze_loader as openvino::ze_loader, so can not just use system ze_loader, still need to find and alias here. From local test, once we install ze_loader and export it to env, build sample still have issue:
`CMake Error at common/utils/CMakeLists.txt:10 (add_library):
  Target "ie_samples_utils" links to target "openvino::ze_loader" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
`